testing/agent-exporter/src/main/java/io/opentelemetry/javaagent/testing/exporter/OtlpInMemoryMetricExporter.java

```diff
@@ -19,11 +21,26 @@
 import java.util.List;
 import java.util.Queue;
 import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.logging.Logger;
 
 class OtlpInMemoryMetricExporter implements MetricExporter {
 
+  private static final Logger logger = Logger.getLogger(OtlpInMemoryMetricExporter.class.getName());
+
   private final Queue<byte[]> collectedRequests = new ConcurrentLinkedQueue<>();
 
+  private final AggregationTemporality aggregationTemporality;
```

Comment:
can this be static, e.g.

```suggestion
  private static final AggregationTemporality aggregationTemporality = initAggregationTemporality();
```